backend/src/router.go

```diff
@@ -220,7 +292,7 @@ func postSignup(c *gin.Context) {
 	}
 
 	hash, _ := bcrypt.GenerateFromPassword([]byte(body.Password), bcrypt.DefaultCost)
-	DB.Create(&User{Username: body.Username, Password: string(hash), FirstName: "Anonymous", LastName: "", Email: "unknown email", Phone: "unknown number"})
+	DB.Create(&User{Username: body.Username, Password: string(hash), FirstName: "", LastName: "", Email: "", Phone: ""})
```

Comment:
Addressed in another PR, so will require a rebase.

Reply:
Yeah, noticed a PR went through about this after I went and changed it.